src/net.h

```diff
@@ -590,6 +603,21 @@ class CNode
         return nRefCount;
     }
 
+    /** Returns true if a transaction with the passed properties will likely get accepted into this node's mempool */
+    bool IsTxAcceptable(const CTxProperties &props)
+    {
+        // Checking the descendants makes no sense -- the target node can't have descendants in its mempool if it
+        // doesn't have this transaction!
+        if (props.countWithAncestors > nLimitAncestorCount)
+            return false;
+        if (props.sizeWithAncestors > nLimitAncestorSize)
+            return false;
```

Comment:
what about descendants, or does it really matter because they should be equal anyway? ...not sure why we even have a number for both descendants and ancestors.

Reply:
Yes, I pointed this out in keybase internal dev if you want look back but basically, the destination node cannot have a descendant count > 0 for this tx because that would mean an orphan is in the mempool.  The descendant check during mempool admission is useless.

And checking descendants doesn't happen anywhere else (and that's a good thing)... if a node somehow got into a state where it has 50 chained tx, does the mempool delete tx 0-25 because they have too many descendants.

If descendants were checked, I could issue a tx that pays someone with change, then quickly spend the change 26 times so that the original tx now gets rejected.